mantis-control-plane/mantis-control-plane-server/src/main/java/io/mantisrx/master/resourcecluster/ResourceClusterActor.java

```diff
@@ -454,6 +454,9 @@ private void onTaskExecutorAssignmentRequest(TaskExecutorAssignmentRequest reque
                 assignmentTimeout);
             sender().tell(matchedExecutor.get().getKey(), self());
         } else {
+            metrics.setCounter(
```

Comment:
I don't see where this is being set? Am I missing something?

Reply:
let me rename to `incrementCounter`. counters can't be "set", they get incremented,